Reverse-Proxy-with-Cloudflare-Tunnel.md

```diff
@@ -43,7 +43,7 @@ Cons:
 
 The steps are actually very simple. However, since the concept is pretty new to anyone, it may be good to write it in detail. 
 
-But trust me, once you learnt, you will remember how to config without this guide again! While for Nginx or Traefik, I never could remember how to config without googling it.
+But trust me, once you learn, you will remember how to config without this guide! While for Nginx or Traefik, I never could remember how to config without googling it.
```

Comment:
I think the wording is still a bit weird

```suggestion
But trust me, once you learn, you will remember how to configure a cloudflare tunnel without this guide! (for Nginx or Traefik, I could never remember how to configure them without googling it)
```

Reply:
> I think the wording is still a bit weird

Even better! Just wanted to point out without making a lot of changes to the doc that was prepared.